### DISCUSSION TITLE
一个任务流中用多个switch节点 后面的不执行

### DISCUSSION THREAD

**Discussion by maketubu7** [apache/dolphinscheduler#14133] [General]
整体流程图
![image](https://github.com/apache/dolphinscheduler/assets/39480244/207e7459-8562-4be0-8f34-e58a02b2db91)
startt2节点设置
![image](https://github.com/apache/dolphinscheduler/assets/39480244/972edd1a-4184-4a6a-a2d0-fa45662c688c)
start2下游的switch节点
![image](https://github.com/apache/dolphinscheduler/assets/39480244/ab1ae0c6-b75a-47ae-b891-bdd03e6cc724)
但是任务执行到start2成功就结束啦，不明白
![image](https://github.com/apache/dolphinscheduler/assets/39480244/1f791b1f-cd6e-4671-aafb-bf108f36de75)

**Comment by fragrance-wang** [comment]
会不会是数据类型的问题，你setValue的时候，设置的是Number类型的数值，判断的时候用String类型判断的
